docs/References.bib

```diff
@@ -147,7 +148,7 @@ @book{Hartle2003gravity
   lccn       = "2002151118",
   year       = "2003",
   publisher  = "Addison-Wesley",
-  url        = "https://www.pearson.com/us/higher-education/program/Hartle-Gravity-An-Introduction-to-Einstein-s-General-Relativity/PGM298455.html",
+  url        = "https://bit.ly/2MTD1Vt",
```

Comment:
Why did you change this?

There is really no need to use a link shortener. If the service stops existing the link will be broken.

Reply:
Because the line was longer than 80 characters, so I couldn't commit it otherwise

Reply:
Thought we had a check that allows committing if it detects the line is a link

Reply:
If we do, it's not currently working. I just checked out the latest develop branch, then made a one character change on a different line in References.bib:

~~~~
$ git diff
diff --git a/docs/References.bib b/docs/References.bib
index cc4175e1..59915355 100644
--- a/docs/References.bib
+++ b/docs/References.bib
@@ -160,7 +160,7 @@
 }
 
   @book{Hartle2003gravity,
-  author     = "Hartle, J.B.",
+  author     = "Hartle, J.~B.",
   title      = "Gravity: An Introduction to {Einstein's} General Relativity",
   isbn       = "9780805386622",
   lccn       = "2002151118",
~~~~

Then I tried to commit and got this error:

~~~~
$ git add docs/References.bib
$ git commit -m "Test change References.bib"
Found lines over 80 characters:
docs/References.bib:100:                        and Duez, Matthew D. and Haas, Roland and Kidder, Lawrence
169:  url        = "https://www.pearson.com/us/higher-education/program/Hartle-Gravity-An-Introduction-to-Einstein-s-General-Relativity/PGM298455.html",
242:  title    = "The analytic theory of fluid disks orbiting the {Kerr} black hole",
~~~~

Am I doing something wrong? Or should I file an issue?

Reply:
URLs should be filtered out from the 80 characters test.

/tools/FileTestDefs.sh states 
>     # This doesn't filter out URLs, but I can't think of a way to do
>     # that without breaking the highlighting.  They only get printed
>     # if there's another problem in the file.

I think the URL exceeding the 80 characters limit triggered because the line with 
`The analytic theory of fluid disks orbiting the {Kerr} black hole`
had 82 characters. Since you fixed that in this PR, it should be possible to commit with the long URL.